CHANGELOG.md

```diff
@@ -42,7 +42,10 @@ different versioning scheme, following the Haskell community's
 * C++ codegen now applies the `--export-attribute` to the `ToString`,
   `FromString`, `ToEnum` and `FromEnum` functions.
 * `import` statements can now end with an optional semicolon.
-
+* File and directory paths on the command line, in response files, or in
```

Comment:
what are response files?

Reply:
gbc response files are inputs that start with "@". If the input file is prefixed with "@", then gbc will treat each line of that file as an input file.